index.ts

```diff
@@ -264,4 +262,5 @@ export default {
   switchCamera,
   raiseMyHand,
   lowerMyHand,
+  ...events,
```

Comment:
Comment about: `export * from './src/events`
Events are not exported with it. Only typings at the moment.

Comment about: ` ...events`
Yeah. I also didn't want to mix it from one side.
But from another side:
- it gives easy access to it. I like libraries when I just import default import and I can see methods, which exists there, without looking docs. (sometimes you can find something that was not mentioned in docs)
- looks like there are no conflicts, because events usually have prefix: `on` 

I can move to different exported object, but not sure it make sense.

Reply:
> Comment about: export * from './src/events
Events are not exported with it. Only typings at the moment.

Not only typings. Also `ZoomEmitter` is exported

> looks like there are no conflicts, because events usually have prefix: on

Ok, let's keep it :)